src/helper_inbox.py

```diff
@@ -1,22 +1,25 @@
-"""Helper Inbox performs inbox messagese related operations."""
+"""Helper Inbox performs inbox messagese related operations"""
 
-from helper_sql import sqlExecute, sqlQuery
 import queues
+from helper_sql import sqlExecute, sqlQuery
 
 
 def insert(t):
+    """Perform an insert into the `inbox` table"""
```

Comment:
These backticks may force sphinx to treat inbox as object in near future (see 5d5fb65, [roles](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html)). I suggest double quotes instead - "inbox".

Reply:
Yes, you can. But indentation error isn't related to the docstring. Read the details of the checks.